workers/generic-worker/gw-codegen/main.go

```diff
@@ -122,6 +123,12 @@ func formatSourceAndSave(sourceCode []byte, sourceFile string) error {
 	// first run goimports to clean up unused imports
 	fixedImports, err := imports.Process(sourceFile, sourceCode, nil)
 	var formattedContent []byte
+
+	// goimports often gets confused about versions based on whatever it finds
+	// in GOPATH, so reset the TC version to the appropriate value.  The version
+	// in the string below will be updated during `yarn release`
+	formattedContent = regexp.MustCompile(`github.com/taskcluster/taskcluster/v[0-9]+/`).ReplaceAll(fixedImports, []byte("github.com/taskcluster/taskcluster/v25/"))
+
```

Comment:
I did mess that up!  It does have an effect when a previous version of the go client is in the gopath, since goimports always chooses the lowest major version it finds (or something like that - I can't replicate it by just go-getting the v24 client).  There's a similar hack in the code to generate the go client.  I'll fix this up, but leave it in to avoid those issues sneaking back in.  It should be harmless when goimports acts right, and fix issues when goimports gets confused.

https://github.com/taskcluster/taskcluster/blob/5f1685d3f0aa9b4cf50258ed8842b7381cbdc3cc/clients/client-go/codegenerator/model/model.go#L162-L166